dht/sharder.hh

```diff
@@ -0,0 +1,116 @@
+/*
+ * Modified by ScyllaDB
```

Comment:
You can drop this line, it's only a requirement of the Apache License when importing code. All of the code here is ours.

Reply:
done.